### PR TITLE
[ci] benchmark react

### PR DESCRIPTION
**Pull request by zhulinJulia24**
1. react benchmark test to use 8 gpu in parallel and add benchmark script test into daily test
2. add llama3.1 into evaluation, function and benchmark test  https://github.com/zhulinJulia24/lmdeploy/actions/runs/10109254518
3. add timecouter into evaluation  https://github.com/zhulinJulia24/lmdeploy/actions/runs/10143171693
4. rename w4a16 into 4bits 
5. fix condition: XX is false condition into not XX
6. remove --cap by using chat-template

add stable testflow
add gptq quantization testcase
change test basic docker image to make test step more simple

### CODE REVIEW

File: autotest/config.yaml [hunk 1]
```diff
@@ -25,7 +27,6 @@ turbomind_chat_model:
     - internlm/internlm2-chat-7b-4bits
     - internlm/internlm2-chat-20b-4bits
     - internlm/internlm-chat-20b
-    - internlm/internlm-xcomposer2-vl-7b
```
Comment (lvhan028) [inline autotest/config.yaml:28]: why remove it?

File: autotest/config.yaml [hunk 2]
```diff
@@ -191,10 +194,43 @@ quatization_case_config:
...
+        - internlm/internlm2_5-7b-chat
+        - internlm/internlm2_5-7b-chat-1m
+        - internlm/internlm2-chat-7b
+        - internlm/internlm2-chat-20b
+        - baichuan-inc/Baichuan2-7B-Chat
+        - 01-ai/Yi-6B-Chat
+        - Qwen/Qwen2-7B-Instruct
+        - Qwen/Qwen2-1.5B-Instruct
+        - Qwen/Qwen1.5-7B-Chat
+        - Qwen/Qwen-7B-Chat
```
Comment (lvhan028) [inline autotest/config.yaml:212]: I think we don't need to check qwen-7b-chat and qwen1.5-7b-chat

File: autotest/config.yaml [hunk 3]
```diff
@@ -191,10 +194,43 @@ quatization_case_config:
...
         - internlm/internlm2-chat-7b
         - 01-ai/Yi-6B-Chat
         - internlm/internlm2-20b
+
+
+longtext_model:
+    - meta-llama/Meta-Llama-3-1-8B-Instruct
+    - meta-llama/Meta-Llama-3-8B-Instruct
+    - meta-llama/Meta-Llama-3-1-70B-Instruct
+    - internlm/internlm2_5-7b-chat
```
Comment (lvhan028) [inline autotest/config.yaml:226]: why not internlm2_5-7b-chat-1m?

**Review by lvhan028 — CHANGES_REQUESTED** [review]

File: autotest/config.yaml (hunk 3 above)
Comment (zhulinJulia24) [reply autotest/config.yaml:217]: > why not internlm2_5-7b-chat-1m?

I add internlm2_5-7b-chat-1m into longtext generation benchmark. Still use internlm2_5-7b-chat in other benchmark test.

**Comment by lvhan028** [comment]
linting error is being taken care of in #1931

File: .github/workflows/evaluate.yml [hunk 4]
```diff
@@ -37,11 +37,11 @@ on:
         required: true
         description: 'Dependency packages, you can also set a specific version'
         type: string
-        default: 'pynvml packaging protobuf transformers_stream_generator transformers'
+        default: 'pynvml packaging protobuf transformers_stream_generator transformers human_eval'
```
Comment (lvhan028) [inline .github/workflows/evaluate.yml:40]: Is `human_eval` a pypi package?

Comment (zhulinJulia24) [reply .github/workflows/evaluate.yml:40]: > human_eval

yes, It used in humaneval dataset's evaluation.

Comment (lvhan028) [reply .github/workflows/evaluate.yml:40]: I tried `pip install human_eval`. But there is no such pypi package

File: autotest/tools/restful/test_restful_chat_workspace.py [hunk 5]
```diff
@@ -34,7 +34,10 @@ def getModelList(tp_num):
 @pytest.mark.parametrize('prepare_environment',
                          getModelList(tp_num=1),
                          indirect=True)
-def test_restful_chat_tp1(request, config, common_case_config, worker_id):
+def test_restful_chat_tp1(config, common_case_config, prepare_environment,
+                          worker_id):
+    if 'llama-3-1' in prepare_environment['model'].lower():
```
Comment (lvhan028) [inline autotest/tools/restful/test_restful_chat_workspace.py:39]: why `llama3.1` is a special case

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**